### DISCUSSION TITLE
Task: Extend the toyModels library - a network Model

### DISCUSSION THREAD

**Discussion by nbecu** [cormas/cormas#777] [Ideas]
Having more ToyModels to test the port to Pharo
   
    8.3 Networks => ??

**Comment by nbecu** [comment]
there is a Demo_Aquifer and a Demo_ArcsNodes in the localDisk model library provided by the current Cormas-Pharo version.
Perhaps those 2 could be candidates for a network toyModel

**Comment by pierbo** [comment]
Yes, I agree.
Note that Demo_Aquifer did evolve recently. See:
[[http://cormas.cirad.fr/en/applica/demoAquifer.htm](http://cormas.cirad.fr/en/applica/demoAquifer.htm)].
Nevertheless, I don't know how to create the Transect and its CrossSectionChart that displays a vertical section of the landscape with the water table changing dynamically.
     Pierre
